contracts/core/liquidators/impl/TwoAssetPriceBoundedLinearAuction.sol

```diff
@@ -92,6 +102,72 @@ contract TwoAssetPriceBoundedLinearAuction is LinearAuction {
         );
     }
 
+    /**
+     * Calculates the minimumBid. First calculates the minimum token flow for the pair at fair value using
+     * maximum natural unit of two Sets. If that token flow is below 1000 units then calculate minimumBid
+     * as such:
+     *
+     * minimumBid = maxNaturalUnit*1000/min(tokenFlow)
+     *
+     * Else, set minimumBid equal to maxNaturalUnit. This is to ensure that around fair value there is ample
+     * granualarity in asset pair price changes and not large discontinuities.
+     *
+     * @param _auction            Auction object
+     * @param _currentSet         CurrentSet, unused in this implementation
+     * @param _nextSet            NextSet, unused in this implementation
+     */
+    function calculateMinimumBid(
+        Auction.Setup storage _auction,
+        ISetToken _currentSet,
+        ISetToken _nextSet
+    )
+        internal
+        view
+        returns (uint256)
+    {
+        // Get full Unit amount and price for each asset
+        AssetInfo memory assetOne = getAssetInfo(_auction.combinedTokenArray[0]);
+        AssetInfo memory assetTwo = getAssetInfo(_auction.combinedTokenArray[1]);
+
+        // Calculate current spot price as assetOne/assetTwo
+        uint256 spotPrice = calculateSpotPrice(assetOne.price, assetTwo.price);
+
+        // Calculate auction price at current asset pair spot price
+        uint256 auctionFairValue = convertAssetPairPriceToAuctionPrice(
```

Comment:
for consistency, maybe name the function `calculateSpotPriceToAssetPrice`? If that is confusing, maybe rename `SpotPrice` to `AssetPairPrice